C7GameData/Player.cs

```diff
@@ -2,71 +2,64 @@
 
 namespace C7GameData
 {
-using System;
-
-public class Player
-{
-	public string guid { get; set; }
-	public int color { get; set; }
-	public bool isBarbarians = false;
-	//TODO: Refactor front-end so it sends player GUID with requests.
-	//We should allow multiple humans, this is a temporary measure.
-	public bool isHuman = false;
-	public bool hasPlayedThisTurn = false;
```

Comment:
This variable shouldn't have been removed; code in TurnHandling.cs depends on it.  I'm pretty sure Development doesn't compile currently due to its removal :(.

It looks like the guid (line 9) also got deleted, but is still in use.  Although there also is a line in GameData.cs where it tries to access the guid as a Guid instead of a string, which means that it doesn't work when I revert line 9.

Most likely due to a bad rebase or merge, but it is a +1 for having an automatic compile check on PRs.

Reply:
hmm, iirc I rebased without conflict from my working branch and assumed it was fine. I'll revert for now and fix this later

Reply:
That is what an automatic PR build would catch though... if we assumed our code wasn't working, we wouldn't think it was ready to merge :).

Reply:
haha yes I agree, we should definitely compile / run tests before merging 😄